conda/common/os/windows.py

```diff
@@ -38,3 +71,25 @@ def is_admin_on_windows():  # pragma: unix no cover
             result = 'unknown'
 
     return result
+
+
+def run_as_admin(cmd_line):
+    """
+    See:
+    - http://stackoverflow.com/a/19719292/1170370 on 20160407 MCS.
+    - msdn.microsoft.com/en-us/library/windows/desktop/bb762153(v=vs.85).aspx
+    """
+    code = None
+    if _ctypes:
+        params = " ".join(['"%s"' % (x, ) for x in cmd_line[1:]])
```

Comment:
This line looks off to me.  Is it really ok to quote every single parameter?  And why the single-value tuple for the string interpolation?

Reply:
> Is it really ok to quote every single parameter?

Well we could do it more conditionally based on the contents, but I think it is ok.